packages/core/src/components/panel-stack/panel-stack.scss

```diff
@@ -0,0 +1,99 @@
+// Copyright 2018 Palantir Technologies, Inc. All rights reserved.
+
+@import "../../common/variables";
+@import "~@blueprintjs/core/src/common/react-transition";
+
+.#{$ns}-panel-stack {
+  position: relative;
+  overflow: hidden;
+
+  .push-#{$ns}-panel-stack-transition-enter {
```

Comment:
$ns must be the first thing in the class. `#{$ns}-panel-stack-transition-push-*`